README.md

```diff
@@ -13,6 +13,10 @@ You can run Kubernetes on Google Cloud Platform in either:
 * Kubernetes on Google Compute Engine virtual machines
 * Google Kubernetes Engine
 
+## Documentation
```

Comment:
Suggestion: instead of making this a new section, add as a paragraph at the end of the `Overview` section. It could say: "Examples", with the link.